.github/workflows/workflow.yml

```diff
@@ -48,12 +48,12 @@ jobs:
     timeout-minutes: 10
     steps:
       - uses: actions/checkout@master
-      - uses: actions/setup-python@master
+      - uses: actions/setup-python@v3
         with:
           python-version: 3.7
-      - uses: actions/setup-go@master
+      - uses: actions/setup-go@v3
         with:
-          go-version: "^1.14.0"
+          go-version: "1.17.7"
```

Comment:
We use different versions throughout this file. That's not desirable. I propose we use `>=1.14.0 <=1.17.0` throughout as this makes explicit the ranges for which it works.

Reply:
Feel free to open an issue for it.